src/NearOrg/Notifications/NotificationsMiddleware.jsx

```diff
@@ -0,0 +1,127 @@
+const { fetchGraphQL, loadItems } = VM.require("${REPL_ACCOUNT}/widget/Entities.QueryApi.Client") || (() => {});
+
+const [shouldFallback, setShouldFallback] = useState(false);
+
+const lastNotificationQuery = `
+  query IndexerQuery {
+    data: dataplatform_near_notifications_notifications( limit: 1, order_by: { block_height: desc }) {
+      blockHeight
+    }
+  }
+`;
+
+const notificationsQuery = `query NotificationsQuery($offset: Int, $limit: Int) {
+  data: dataplatform_near_notifications_notifications(
+    distinct_on: blockHeight,
```

Comment:
Hey @gabehamilton! According to our private discussion, I tried to use two fields here like so:
```suggestion
    distinct_on: [blockHeight, initiatedBy],
```
but also I should update `order_by` like so
```
    order_by: {blockHeight: desc, initiatedBy: asc},
```

but after this steps I received a list of data which is ordered in a very strange way. Am I missed something?

Reply:
Changing `initiatedBy: asc` => `initiatedBy: desc` doesn't help

Reply:
I see the same behavior in the GraphiQL editor. It looks like the `initiatedBy` order is taking precedence, regardless of order in the query.

Reply:
Ah, order by should be `order_by: [{blockHeight: desc}, {initiatedBy:asc}]`